### ISSUE TITLE
kube-router: rbac.authorization.k8s.io/v1beta1 deprecated

### ISSUE THREAD

**Issue by gclawes** [enix/helm-charts#71]
Per `kubectl` warning:
```
Warning: rbac.authorization.k8s.io/v1beta1 ClusterRoleBinding is deprecated in v1.17+, unavailable in v1.22+;
```

The RBAC (`ClusterRole` and `ClusterRoleBinding`) in kube-router is using these deprecated APIs, should change to `rbac.authorization.k8s.io/v1b` in order to support new cluster versions.

**Comment by npdgm** [comment]
Hi @gclawes,
A new release of the chart is available with a fix starting with version 1.5.3 (kube-router v1.3.2).
Thanks for reporting this issue.
Cheers

**Comment by gclawes** [comment]
Awesome, thanks!
